test/index.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+const babelPresetEnv = require("../lib/index.js");
```

Comment:
Not a big deal, if we get everything working we can also just move it into babel/babel